eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -230,7 +230,7 @@ void failsOnCritical(@TempDir final Path temp) throws IOException {
                     "    TRUE > x",
                     "    FALSE > x"
                 ).with("trackOptimizationSteps", true)
-                .execute(new FakeMaven.Optimize())
+                .execute(new FakeMaven.Verify())
```

Comment:
@c71n93 I think this whole test does not belong to the class. Here we should test `OptimizeMojo`, not `VerifyMojo`. Let's move the test to the `VerifyMojoTest`. But here I believe we should add the test (if it does not exist yet) that would check that program with errors DOES NOT fail on optimization step

Reply:
@maxonfjvipon done